pkg/release/annotator.go

```diff
@@ -89,19 +89,9 @@ func releaseManifestToUnstructured(manifest string, logger log.Logger) []unstruc
 	manifests := releaseutil.SplitManifests(manifest)
 	var objs []unstructured.Unstructured
 	for _, manifest := range manifests {
-		bytes, err := yaml.YAMLToJSON([]byte(manifest))
-		if err != nil {
-			logger.Log("err", err)
-			continue
-		}
-
-		if len(bytes) == 0 {
-			continue
-		}
-
 		var u unstructured.Unstructured
-		if err := u.UnmarshalJSON(bytes); err != nil {
-			logger.Log("err", err)
+
+		if err := yaml.Unmarshal([]byte(manifest), &u); err != nil {
```

Comment:
Change to just attempt unmarshalling directly to the Unstructured type and ignore if there are any issues.